src/transformers/models/time_series_transformer/modeling_time_series_transformer.py

```diff
@@ -1316,9 +1437,9 @@ def forward(
             # [bsz, seq_len] -> [bsz, 1, tgt_seq_len, src_seq_len]
             encoder_attention_mask = _expand_mask(encoder_attention_mask, inputs_embeds.dtype, tgt_len=input_shape[-1])
 
-        hidden_states = inputs_embeds
-        hidden_states = self.layernorm_embedding(hidden_states)
-
+        hidden_states = self.value_embedding(inputs_embeds)
+        embed_pos = self.embed_positions(inputs_embeds.size(), past_key_values_length=self.config.context_length)
```

Comment:
Could we make this PR backwards compatible by adding boolean arguments to turn on/off the input projection layer and position embeddings?